.github/workflows/contribs.yml

```
@@ -0,0 +1,23 @@
+name: Contribs
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+jobs:
+  contribs:
+    runs-on: ubuntu-latest
+    name: Contribs
+    steps:
+      - name: Contribs
+        uses: carlescufi/action-contribs@main
+        with:
+          github-token: ${{ secrets.NCS_GITHUB_TOKEN }}
+          command: 'external'
+          messages: |
+                    Thank you for your contribution!
+                    It seems you are not a member of the nrfconnect GitHub organization.
+                    We are currently not accepting external contributions, but this may change soon.
+                    Please visit https://devzone.nordicsemi.com/ for further information.
```

We don't have information on this on DevZone, or do we?
Or did you mean something like:
```suggestion
                    Please visit https://devzone.nordicsemi.com/ to raise any issues you found or ask for help.
```